exercises/bob/example.go

```diff
@@ -2,29 +2,32 @@ package bob
 
 import "strings"
 
-const testVersion = 3
-
-func Hey(drivel string) string {
-	switch drivel = strings.TrimSpace(drivel); {
-	case silent(drivel):
+// Hey returns Bob's responses to any given dialogue
+func Hey(dialogue string) string {
```

Comment:
`drivel` felt a little negative, so changed it to the much more mundane `dialogue` :woman_shrugging: